ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/VulnerabilityStateTabs.tsx

```diff
@@ -42,10 +44,12 @@ function VulnerabilityStateTabs({
             <Tab
                 eventKey="OBSERVED"
                 title={<TabTitleText>{titleOverrides?.observed ?? 'Observed'}</TabTitleText>}
+                tabContentId={vulnStateTabContentId}
```

Comment:
I was a little confused about this one. Should the `tabContentId` for each be unique?

Reply:
In the case of each tab having it's own target container, I would make each one unique to that container. In this case though, each tab drives the content for just a single container element so we don't have three different values we can use.

Reply:
Somewhat similar to need for only one `id` in #12238